geojson_modelica_translator/system_parameters/schema.json

```
@@ -29,6 +29,10 @@
             }
           ]
         },
+        "weather": {
+          "description": "Path, relative or absolute, to a weather file for the district",
```

should we be explicit on the filetype? MOS vs EPW.

---

I think we should say ...

```suggestion
          "description": "Path, relative or absolute, to a weather file for the district in Modelica format (MOS).",
```